config/jest.setup.js

```diff
@@ -1,3 +1,4 @@
+require('whatwg-fetch');
```

Comment:
Why would a fetch shim be needed in jest?

Reply:
@himdel you need this to mock api requests